server/GRPC_TA2_TA3/ta3ta2_api/core.proto

```diff
@@ -0,0 +1,252 @@
+syntax = "proto3";
```

Comment:
Is this an error of adding files under ta3ta2_api to git? They are fetched and do not need to be committed.